src/domain/proposal.ts

```diff
@@ -134,6 +134,13 @@ export function updateGPProposal(
   proposal.organizationId = gpProposal.value0;
   proposal.expiresInQueueAt = timestamp.plus(params.value1);
   proposal.createdAt = timestamp;
+  updateThreshold(
+    proposal.dao.toString(),
+    gp.threshold(proposal.paramsHash, proposal.organizationId),
```

Comment:
proposal.organizationId seems to be redundant. as it is the id of GPQUE.

Reply:
It causes problems to remove it and Alchemy might be using it.